docs/src/snap/howto/cos-lite.md

```diff
@@ -0,0 +1,142 @@
+# Integrating with COS Lite
+
+**Canonical Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between Canonical Kubernetes and COS Lite (Canonical
+Observability Stack). This guide will help you integrate a COS Lite
+deployment with a Canonical Kubernetes deployment.
+
+This document assumes you have a controller with an installation of
+Canonical Kubernetes. If this is not your case, refer to
+["Installing Canonical Kubernetes"][how-to-install].
+
+## Preparing a platform for COS Lite
+
+First, create a Microk8s model to act as a deployment cloud for COS Lite:
+
+```
+juju add-model --config logging-config='<root>=DEBUG' microk8s-ubuntu aws
```

Comment:
```suggestion
juju add-model --config logging-config='<root>=DEBUG' microk8s-ubuntu
```
let's not specify the cloud here (juju will either use the current cloud or prompt you for the data it needs); if the user has trouble adding this model, they can learn more in the juju-models docs linked below.